trail/upload/process_uploads/fits_processor.py

```diff
@@ -331,8 +331,8 @@ def process(self):
         else:
             standardizedResult.extendThumbnails(tmpResult)
 
-        if len(standardizedResult.thumbnails) != len(standardizedResult.wcs):
-            raise RuntimeError("Can not unambiguously assign thumbnails to WCSs!")
+        # if len(standardizedResult.thumbnails) != len(standardizedResult.wcs):
+        #     raise RuntimeError("Can not unambiguously assign thumbnails to WCSs!")
 
         for thumb, wcs in zip(standardizedResult.thumbnails, standardizedResult.wcs):
```

Comment:
This is then also not correct. We would have to check if `thumbnails` is a list or not and if it is then it must be the same length, otherwise we need to go and assign each wcs the same `thumbnails` because they are all pasted in a single image.